src/main/resources/db/migration/V12_Totrinnskontroll.sql

```diff
@@ -0,0 +1,16 @@
+CREATE table totrinnsstatus
+(
+    id             UUID PRIMARY KEY,
+    behandling_Id UUID NOT NULL REFERENCES behandling (id),
```

Comment:
Småpirk, men fint å kjøre en formattering :) 
```suggestion
    behandling_Id UUID         NOT NULL REFERENCES behandling (id),
```